images/jdk/tests/main.tf

```diff
@@ -13,7 +13,8 @@ data "oci_exec_test" "version" {
   script = "docker run --rm $IMAGE_NAME javac -version"
 }
 
-data "oci_exec_test" "hello-world" {
-  digest = var.digest
-  script = "${path.module}/02-hello-world.sh"
+module "jre-test" {
+  source    = "../../jre/tests"
+  digest    = var.digest
+  sdk-image = var.digest
```

Comment:
```suggestion
  sdk-image = var.sdk-image
```

Reply:
Should the tests have passed like this?

Reply:
Yes because you are incorrect.  The JDK is both the runtime and the SDK here